fog/view/enclave/impl/src/lib.rs

```diff
@@ -352,10 +360,31 @@ where
         )
     }
 
+    // Takes each MultiViewStoreResponseQuery
     fn get_minimum_highest_processed_block_count(
-        &self,
-        _shard_query_responses: Vec<(QueryResponse, BlockRange)>,
+        responses: &[DecryptedMultiViewStoreQueryResponse],
     ) -> u64 {
-        todo!()
+        let responses_with_gaps = responses
```

Comment:
:+1: Really digging the isolated logic of this function. It's easy to reason about in isolation without worrying about the other logic in creating the response